README.MD

```diff
@@ -9,15 +9,6 @@ This bundle provides two endpoints:
 `/ping` for general health check
 `/health-check` for more detailed health check for configured services.
 
-To configure the services to be checked, you can add the following configuration to your `zu_health_check.yaml` file:
-
-```yaml
-zu_health_check:
-    type:
-        smtp: false
-        doctrine: false
-```
-
```

Comment:
Just noticed that this should remain inside the README.

Reply:
It is in there twice. I left it in the configuration section